src/content/docs/cheerpj3/11-guides/CheerpJ-explanation.mdx

```diff
@@ -0,0 +1,81 @@
+
+1. The Java byte code runs within an interpreter.
+2. Then this byte code is Just-In-Time compiled to optimized JavaScript.
+
+The interpreter does not only deal with initialization and rarely-used code, but also gathers necessary information for JIT-ting. The generated code is very efficient, and the internal optimizer can, among other things, inline and devirtualize calls, which is extremely important for a language such as Java.
+
+![CheerpJ 3.0 JRE diagram](/cheerpj3/assets/JRE.png)
+
+Alongside being able to run Java applications in the browser, CheerpJ runtime also provides advanced, bidirectional Java-JavaScript interoperability. Meaning you can access DOM from Java by [implementing `native` methods directly in JavaScript](/cheerpj3/guides/Implementing-Java-native-methods-in-JavaScript). You can also interact with Java methods, objects and arrays directly from JavaScript by using the new [`cheerpjRunLibrary` API](/cheerpj3/reference/cheerpjRunLibrary).
+
+### The virtualized window manager
+
+What would be of an application without its graphical UI? CheerpJ's window manager supports Java AWT/Swing by converting windows to a hierarchy of HTML elements and HTML5 canvases.
+
+Swing applications, will render exactly as they do in native. Swing Look&Feel is also supported, including 3rd party ones. Multi-window applications are supported, with keyboard focus being managed as expected. Integration with the system clipboard can be enabled via an initialization option.
+
+### The virtualized file system
+
+CheepJ provides [multiple filesystem backends](cheerpj3/guides/File-System-support) to accommodate different application needs, including access to server-hosted files and persistent local storage.
+
+![](/cheerpj2/assets/filesystem.png)
+
+You can learn more about CheerpJ's filesystem at our [files and filesystems guide].
+
+### Networking support
+
+For same-origin HTTP/HTTPS requests, CheerpJ will be able to transparently use `fetch`. More generalized networking is supported via Tailscale, a VPN technology using WebSockets as a transport layer. It can support many different networking scenarios, including access to private network services, peer-to-peer connections between users and access to the wider internet via a user/application provided *exit node*.
+![CheerpJ 3.0 general networking](/cheerpj3/assets/general_networking.png)
+You can learn more about Networking with CheerpJ and Tailscale at our [Networking guide] and our [Networking tutorial 1] and [Networking tutorial 2].
+
+## Use cases
+
+Here we provide a list of the most common uses-cases that CheerpJ supports:
+
+- Restoration of legacy gaming, educational or personal content.
+- Modernization of enterprise software such Java applications, applets, Web Start / JNLP.
+- Building web applications with Java libraries.
+- Fully client-side Java development environments and playgrounds.
+
+CheerpJ is free for personal use and technical evaluation. See [licensing] for details.
```

Comment:
This doesn't feel like it fits here, it's not really an explanation. Maybe a whole separate use cases page where we go into more detail? We could also talk about actual clients (linking to the [case studies](https://leaningtech.com/case-studies/)) and migrate the community project list to this page too